### PR TITLE
Feat: Add new model to model hub

### PR DESCRIPTION
**Pull request by hahuyhoang411**
## Describe Your Changes

Add 1-3B models with good coherence.

## Self Checklist

- [x] Stable Zephyr 3B
- [x] Delete Yarn Mistral (issue: https://github.com/janhq/jan/issues/1549)
- [x] Dolphin Phi-2 (Thank you @tomasmcm  https://github.com/janhq/jan/pull/1603)
- [ ] ~Nous-Hermes mixtral (https://huggingface.co/NousResearch/Nous-Hermes-2-Mixtral-8x7B-DPO#prompt-format)~
- [x] LlamaCorn-1.1B (https://huggingface.co/jan-hq/LlamaCorn-1.1B)
- [ ] ~Stable Code 3B (https://huggingface.co/stabilityai/stable-code-3b)~
- [ ] Update model-extension's version

### CODE REVIEW

File: models/stable-zephyr-3b/model.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "source_url": "https://huggingface.co/TheBloke/Starling-LM-7B-alpha-GGUF/resolve/main/starling-lm-7b-alpha.Q4_K_M.gguf",
+    "id": "stable-zephyr-3b",
+    "object": "model",
+    "name": "Stable Zephyr 3B Q8",
+    "version": "1.0",
+    "description": "StableLM Zephyr 3B is trained on a mix of publicly and synthetic datasets. It's focus on safety and reliability.",
+    "format": "gguf",
+    "settings": {
+      "ctx_len": 4096,
+      "prompt_template": "<|user|>{prompt}<|endoftext|><|assistant|>"
```
[the remark this answers is not in the document]
Comment (hahuyhoang411) [reply models/stable-zephyr-3b/model.json:11]: ah yes, it was my mistake. Thank you for pointing out

**Comment by tomasmcm** [comment]
Not many people have 30GB of RAM available to run the Mixtral models 🙈 Can we merge this without that? Having stable-zephyr and dolphin-phi would already be a great improvement.

**Review by louis-jan — CHANGES_REQUESTED** [review]
Just a reminder, let's update model-extension's version

**Comment by hahuyhoang411** [comment]
Prepare to merge with https://github.com/janhq/jan/pull/1659

**Pull request closed without merging**